docs/guide/breadcrumbs/breadcrumbs_api/postsilvermodulebreadcrumbsgenerator.md

```diff
@@ -1,40 +1,36 @@
 # PostSilverModuleBreadcrumbsGenerator
 
-This breadcrumbs generator is for requests after a silver.module, which process the controller of the previous silver.module.
+`Silversolutions\Bundle\EshopBundle\Service\Breadcrumbs\PostSilverModuleBreadcrumbsGenerator`
+is for requests after a silver.module, which process the controller of the previous silver.module.
 
-**Example**: A silver.module eZ content object specifies the controller SisoCustomerCenterBundle:Form:form in order to display a form. After this form is submitted, it will not request the URL of the silver.module, but the URL of the controller, defined by its route. But nonetheless, the breadcrumbs of the former silver.module shall be displayed. 
+**Example**: A silver.module Content item specifies the controller `SisoCustomerCenterBundle:Form:form` in order to display a form. After this form is submitted, it will not request the URL of the silver.module, but the URL of the controller, defined by its route. But nonetheless, the breadcrumbs of the former silver.module shall be displayed. 
 
-This generator is responsible for this special case of requests. It listens to two kernel events in order to add some request attributes and save the last used silver.module ID temporarily in the session.  
-Hint
+This generator is responsible for this special case of requests. It listens to two kernel events in order to add some request attributes and save the last used silver.module ID temporarily in the session.
 
-silver.modules themselves are standard eZ content and their breadcrumbs are handled solely by the parent of this class which is [EzContentBreadcrumbsGenerator](ezcontentbreadcrumbsgenerator.md).
+!!! tip
 
-## FQN
+    silver.modules themselves are standard Content items and their breadcrumbs are handled solely by the parent of this class which is [`EzContentBreadcrumbsGenerator`](ezcontentbreadcrumbsgenerator.md).
 
-Silversolutions\Bundle\EshopBundle\Service\Breadcrumbs\PostSilverModuleBreadcrumbsGenerator
-
-## Criterion for responsibility
+## When it is used
 
 The generator triggers if the session attribute 'silverModuleLocation' is set. This is set by an event listener callback, which is described below.
 
-## Notes to the rendering process
-
-In order to memorize a formerly displayed silver.module element, its ID is stored in the session and restored if the next page matches the controller, which is defined in the respective silver.module. This is achieved via two kernel event listeners:
-
-### Event: kernel.controller
+## Rendering process
 
-Method: Silversolutions\Bundle\EshopBundle\Service\Breadcrumbs\PostSilverModuleBreadcrumbsGenerator::onKernelRequest()
+In order to memorize a formerly displayed silver.module element, its ID is stored in the session and restored if the next page matches the controller, which is defined in the respective silver.module. This is achieved via two kernel event listeners.
```

Comment:
Who/what momerizes the silver.module? I'd rewriite the sentence in active voice. Also, I'd break it in a series of shorter sentences. I would spell out the object hidden behind "It".